slices/python3.8-minimal.yaml

```diff
@@ -0,0 +1,13 @@
+package: python3.8-minimal
+
+slices:
+  core:
+    essential:
+      - libc6_libs
+      - libpython3.8-minimal_libs
+      - libexpat1_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/python3.8:
+      /usr/local/lib/python3.8/: {make: true, mode: 02775}
```

Comment:
if this from the maintainer scripts? If so, out of curiosity, could the same be done in starlark?